src/framework/components/particlesystem/particlesystem_system.js

```diff
@@ -195,12 +195,49 @@ pc.extend(pc, function() {
                 type: 'boolean',
                 defaultValue: false
             }, {
-                name: "spawnBounds",
-                displayName: "Spawn Bounds",
-                description: "Defines an AABB in which particles are allowed to spawn",
+                name: "emitterShape",
+                displayName: "Emitter Shape",
+                description: "Defines a shape in which particles are allowed to spawn",
+                type: "enumeration",
+                options: {
+                    enumerations: [{
```

Comment:
I don't like the sound of this. The shape affect how particles behave. It has to have a shape. I'd probably default it to a sphere with 0 radius.